apps/builder/app/canvas/features/text-editor/text-editor.tsx

```diff
@@ -113,6 +114,29 @@ const CaretColorPlugin = () => {
   return null;
 };
 
+const OnChangeOnBlurPlugin = ({
+  onChange,
+}: {
+  onChange: (editorState: EditorState) => void;
+}) => {
+  const [editor] = useLexicalComposerContext();
+  const handleChange = useEffectEvent(onChange);
+
+  useEffect(() => {
+    const handleBlur = () => {
+      handleChange(editor.getEditorState());
+    };
+
+    // https://github.com/facebook/lexical/blob/867d449b2a6497ff9b1fbdbd70724c74a1044d8b/packages/lexical-react/src/LexicalNodeEventPlugin.ts#L59C12-L67C8
+    return editor.registerRootListener((rootElement, prevRootElement) => {
+      rootElement?.addEventListener("blur", handleBlur);
+      prevRootElement?.removeEventListener("blur", handleBlur);
+    });
+  }, [editor, handleChange]);
+
+  return null;
```

Comment:
I am guessing plugin types forced you to return null

Reply:
All plugins we have returns null. All official lexical plugins returns null. So do I